rsyslog/init.sls

```diff
@@ -1,9 +1,10 @@
 {%- if pillar.rsyslog is defined %}
 include:
 {%- if pillar.rsyslog.server is defined %}
```

Comment:
is defined = include state (so it's correct)
enabled = determine what to do inside the state (eg. setup everything vs. disable service)